src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
This is the place I'm least comfortable with. On the one hand, I believe it would be better to have dedicated responsibilities injected (`IModifyExtend` xor `IModifyUpdate`) into different interfaces. Thus, the capability of the Rocketlet could be understood from the implemented interfaces. On the other hand I fear overengineering. By parsing the code, you could also figure out what's actually accessed.

WDYT?